internal/access/signup.go

```diff
@@ -64,6 +64,7 @@ func Signup(c *gin.Context, name, password string) (*models.Identity, error) {
 		Subject:   uid.NewIdentityPolymorphicID(identity.ID),
 		Privilege: models.InfraAdminRole,
 		Resource:  "infra",
+		CreatedBy: models.CreatedBySystem,
```

Comment:
This is a no-op change, because `models.CreatedBySystem` has a value of `0`, but I added this here and in another place to be more explicit about the value. And so that if we change the value in the future (we might in #1660), then it'll be correct.

I wonder if we actually want to use a non-zero value for this, so that we van error if `CreatedBy` is set to 0, to catch bugs in the future. Although I'm not sure how we ensure that `uid.New` doesn't generate the same ID.

Reply:
There was some weirdness around the zero value of uid.ID (unintentionally from my point of view) defaulting to 1 instead of 0. Not sure what effect that would have here. I'd probably like CreatedBySystem to have a value > 1 and a validation rule that doesn't accept <= 1